Multisig/UI/Transaction/TransactionDetailsViewController/TransactionDetailsViewController.swift

```diff
@@ -76,31 +79,71 @@ class TransactionDetailsViewController: LoadableViewController, UITableViewDataS
 
     // MARK: - Signing
 
-    fileprivate func configureConfirmButton() {
-        // confirm button sticks to the bottom of the screen
+    fileprivate func configureActionButtons() {
+        // Actions Container View sticks to the bottom of the screen
         // and is on top of the table view.
         // it is shown only when table view is shown.
 
+        actionsContainerView = UIStackView()
```

Comment:
Maybe it is time to create a separate XIB with both buttons and encapsulate all of it there?

Reply:
I'm up to create XIB for whole of transaction details screen, but this need another refactor